portal/src/graphql/portal/AdminAPIConfigurationScreen.tsx

```diff
@@ -32,10 +35,15 @@ import ScreenLayoutScrollView from "../../ScreenLayoutScrollView";
 import TextField from "../../TextField";
 import PrimaryButton from "../../PrimaryButton";
 import ActionButton from "../../ActionButton";
+import ButtonWithLoading from "../../ButtonWithLoading";
+import DefaultButton from "../../DefaultButton";
+import { useUpdateAppAndSecretConfigMutation } from "./mutations/updateAppAndSecretMutation";
 
 interface AdminAPIConfigurationScreenContentProps {
   appID: string;
   queryResult: AppAndSecretConfigQueryResult;
+  generateKey: () => void;
+  showDeleteKeyDialog: (keyID: string) => void;
```

Comment:
Handle confirmation dialog in AdminAPIConfigurationScreenContent. So the new props should be 

```
interface AdminAPIConfigurationScreenContentProps {
  generateKey: () => void;
  deleteKey: (keyID: string) => void;
}
```